DataParTemplate.md

```diff
@@ -77,8 +77,8 @@ This is to allow for valid JSONs. The conversion is carried out internally.
 | ------------------------------------- |:---------------------------------------------:|:------------------:|
 | x.Q.bUseBasilQuantification           | True for using BASIL quantification in addition to ExploreASL's quantification. |  |
 | x.Q.Lambda                            | Brain/blood water coefficient (mL 1H/ mL blood). Example: `0.32` (for GSP phantom). | OPTIONAL, DEFAULT = 0.9 |
-| x.Q.T2art                             | `T2*` of arterial blood at 3T, only used when no M0 image (ms). | OPTIONAL, DEFAULT = 50 |
-| x.Q.BloodT1                           | T1 relaxation time of arterial blood (ms). Defaults (Alsop MRM 2014), 1800 for GSP phantom. | OPTIONAL, DEFAULT = 1650 @ 3T, 1350 @ 1.5 T |
+| x.Q.T2art                             | `T2*` of arterial blood, only used when no M0 image (ms). | OPTIONAL, DEFAULT = 50 @ 3T|
+| x.Q.BloodT1                           | T1 relaxation time of arterial blood (ms). Defaults (Alsop MRM 2014), 1800 for GSP phantom. | OPTIONAL, DEFAULT = 2100 @ 7T, 1650 @ 3T, 1350 @ 1.5 T |
```

Comment:
Should we add references here, now that you can still remember them?

Reply:
That's the point - I don't have those - I took all from OSIPI TF4.1, but some other references were a bit unclear.

Reply:
We have too many values now - I list here only the 3T default and I give the correct values and all references in the Quantify-wrapper where I really used them

Reply:
OK, I'll make a separate issue on adding references to our scripts.